### PR TITLE
UI fixes and updates

### PR DESCRIPTION
**Pull request by jrgarou**
- Moved styles from index.html to `app.scss` @bent0b0x is this fine for embed? It looks ok.
- Wrap the **video** and the **descritption Card**(no info yet)  on `/play` - the `@media` rules gets both them
- Play icon on `VideoFormInfoBox`
image
- `VideoListItem` with @pedrocasa design: thumb, truncate title, status and a small bar(`<VideoProgressBar ... small />`. 
image

- `VideoListItem.onClick` Scroll to `VideoForm` when `VideoForm.offsetTop > window.innerHeight / 2` .

- Updated header and footer size.
- Unbold the headers(`<h1>,<h2>...`)

### CODE REVIEW

File: src/scripts/components/Play.js [hunk 1]
```diff
@@ -540,86 +565,93 @@ class Play extends Component<Props, State> {
     } else {
       return (
         <Wrapper isEmbed={isEmbed}>
-          <PlayerWrapper
```
Comment (bent0b0x) [inline src/scripts/components/Play.js]: This big diff is kind of scary, but if the new player works it works!

Is this mostly because the player is now wrapped in `VideoWrapper`?

File: src/styles/app.scss [hunk 2]
```diff
@@ -14,4 +14,30 @@ html, body, #root {
...
+  height: 50px;
+  left: 50%;
+  position: fixed;
+  top: 50%;
+  transform: translate3d(-50%, -50%, 0);
+  width: 50px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: 400;
```
Comment (bent0b0x) [inline src/styles/app.scss]: I don't think this is going to work for embed. `embed/index.js` imports a separate `/embed/index.scss` file, not this one

File: src/scripts/components/VideoListItem.js [hunk 3]
```diff
@@ -76,6 +107,13 @@ class VideoListItem extends Component<Props, void> {
   }
 
   handleClick () {
+    const form = document.getElementById('video-form')
+    const offsetTop = form.offsetTop - 20
+    if (form) {
+      if (offsetTop > window.innerHeight / 2) {
+        document.getElementById('root').scrollTo(0, offsetTop)
+      }
+    }
```
Comment (bent0b0x) [inline src/scripts/components/VideoListItem.js]: This works, but there is a more react-way to get references to actual dom nodes 😄 (requires no hard-coded IDs).

Take a look at [this line](https://github.com/Paratii-Video/paratii-portal/blob/460fa75750281b9b66a0ad93a4d92dd3e9c72066/src/scripts/components/widgets/VolumeBar.js#L107) in `VolumeBar.js`. Here, we take advantage of `styled-components`'s [innerRef](https://github.com/styled-components/styled-components/blob/master/docs/tips-and-tricks.md#refs-to-dom-nodes) property. This provides a mechanism for us to access the [ref property](https://reactjs.org/docs/refs-and-the-dom.html) on the underlying react elements.

Long story short: `ref/innerRef` are properties that take a callback. This callback is invoked with an `HTMLElement` that you can then cache/store somewhere. `VolumeBar.js` is a good example.

Let me know if you have any questions about how this should work!

File: src/scripts/components/VideoListItem.js [hunk 4]
```diff
@@ -76,6 +107,14 @@ class VideoListItem extends Component<Props, void> {
...
 
   handleClick () {
+    const form = document.getElementById('video-form')
+    if (form) {
+      const offsetTop = form.offsetTop - 20
+
+      if (offsetTop > window.innerHeight / 2) {
+        document.getElementById('root').scrollTo(0, offsetTop)
+      }
+    }
```
Comment (bent0b0x) [inline src/scripts/components/VideoListItem.js]: This works, but there is a more react-way to get references to actual dom nodes 😄 (requires no hard-coded IDs).

Take a look at this line in VolumeBar.js. Here, we take advantage of styled-components's innerRef property. This provides a mechanism for us to access the ref property on the underlying react elements.

Long story short: ref/innerRef are properties that take a callback. This callback is invoked with an HTMLElement that you can then cache/store somewhere. VolumeBar.js is a good example.

Let me know if you have any questions about how this should work!

File: src/styles/app.scss (hunk 2 above)
Comment (jrgarou) [reply src/styles/app.scss]: @bent0b0x, could we import all styles from `src/styles/app.scss`?

File: src/scripts/components/Play.js (hunk 1 above)
Comment (jrgarou) [reply src/scripts/components/Play.js]: @bent0b0x yes. Basically it's just about the wrapper from `/play` and `/embed`

File: src/scripts/components/Play.js [hunk 5]
```diff
@@ -540,86 +565,93 @@ class Play extends Component<Props, State> {
...
+                          requestFullscreen(this.wrapperRef)
+                        } else {
+                          requestCancelFullscreen()
+                        }
+                      }}
+                    />
+                  </OverlayWrapper>
+                )}
+              </Transition>
+              <Player id="player" />
```
Comment (eliawk) [inline src/scripts/components/Play.js]: this `player id="player"` is still there hardcoded? is it correct @bent0b0x?

File: src/scripts/components/VideoListItem.js (hunk 4 above)
Comment (jrgarou) [reply src/scripts/components/VideoListItem.js]: I changed this code in https://github.com/Paratii-Video/paratii-portal/pull/324/commits/acceeb58b913e65813af3ed20a546c5fe1a4aff0. I'm trying to get the reference of `VideoForm` on `VideoListItem` from `VideoManagerContainer`.

When I [click on](https://github.com/Paratii-Video/paratii-portal/blob/acceeb58b913e65813af3ed20a546c5fe1a4aff0/src/scripts/components/VideoListItem.js#L109-L112) a `VideoListItem` the `this.props.videoFormRef` returns `null`. After somes clicks it get the `div.Card`.
if you click on a `VideoListItem` then click on `ADD ANOTHER VIDEO` then again clicks on `VideoListItem` and keep clicking on it(no change to other `VideoListItem`) the `this.props.videoFormRef` will always returns `null`:
getref

File: src/scripts/components/Play.js (hunk 5 above)
Comment (bent0b0x) [reply src/scripts/components/Play.js]: This is needed because of `clappr`. I don't think we have a feasible way around this right now

File: src/styles/app.scss (hunk 2 above)
Comment (bent0b0x) [reply src/styles/app.scss]: You can update `embed/index.js`. Just note that we want to keep `/embed` light so if there things in here that the embed doesn't need let's not import them. Otherwise, import what is needed into `/embed/index.scss`

File: src/scripts/components/VideoListItem.js [hunk 6]
```diff
@@ -76,6 +107,7 @@ class VideoListItem extends Component<Props, void> {
   }
 
   handleClick () {
+    console.log(this.props)
```
Comment (bent0b0x) [inline src/scripts/components/VideoListItem.js]: remove log

File: src/styles/app.scss (hunk 2 above)
Comment (jrgarou) [reply src/styles/app.scss]: Done https://github.com/Paratii-Video/paratii-portal/pull/324/commits/2800e1ddba828514e8a922e32a7618f0b3acab8e

File: src/scripts/components/VideoListItem.js (hunk 6 above)
Comment (jrgarou) [reply src/scripts/components/VideoListItem.js]: done https://github.com/Paratii-Video/paratii-portal/pull/324/commits/fd247c6ae5a0a1a4e0a5e563e6638cd8c96f3dd6

**Comment by jellegerbrandy** [comment]
@jrgarou, could you fix the conflicts, so I can merge this? (go to your branch and do `git merge dev`)
